openapi.yml

```diff
@@ -950,6 +972,9 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/DescriptiveValue"
+        standard:
+          #description: Descriptive or content standards to which the value conforms.
```

Comment:
Cruft?

Reply:
No, you can't provide description with a $ref. So this is just an annotation for humans.

Reply:
OK. Recommend `# description` to make it seem a bit less crufty but not a big deal.